LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Do we have any default license to be used?

Reply:
I would stick with Apache 2, like we do with the other code we created